src/main/application/services/ping/PingUseCase.js

```diff
@@ -1,11 +1,28 @@
+import {inject} from '../../../core/ioc/ioc'
 import {PingReturn} from '../../../domain/ping/PingReturn.js'
+import {CmpStatusRepository} from '../../../domain/status/CmpStatusRepository'
+import {DisplayStatusRepository} from '../../../domain/status/DisplayStatusRepository'
+
+export class PingUseCase {
+  /**
+   *
+   * @param {Object} param
+   * @param {CmpStatusRepository} param.cmpStatusRepository
```

Comment:
You should remove line 9?

Reply:
We need for reference according to JSDoc